scripts/neighbor_advertiser

```diff
@@ -384,10 +384,26 @@ def save_as_json(obj, file_path):
         json.dump(obj, outfile, sort_keys = True)
 
 
+def find_mirror_table_name():
+    acl_tables = config_db.get_keys("ACL_TABLE")
+    v4_table, v6_table = "", ""
+    for table in acl_tables:
+        if MIRROR_ACL_TABLE_NAME in table and MIRROR_ACL_TABLEV6_NAME not in table:
+            v4_table = table
+        elif MIRROR_ACL_TABLEV6_NAME in table:
+            v6_table = table
```

Comment:
If you test MIRROR_ACL_TABLEV6_NAME first, then you don't need 2 conditions for v4_table name, right?